change/@internal-react-composites-64003277-fb49-4fbc-bc6e-1d122ee02dc6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Should this be tagged as pre-release since this work is related to PSTN and 1:N?

Reply:
This should have been `patch`, or even `none` as it has no user-visible impact.

`minor` means that there is new API / large new UX for the customer to use.